org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/WorkspaceSymbolHandler.java

```diff
@@ -54,7 +55,7 @@ public void acceptTypeNameMatch(TypeNameMatch match) {
 						if (match.getType().isBinary()) {

```

Comment:
Wouldn't it be much simpler to just call JDTUtils.toLocatoin(match.getType())? We are using that method in many places, so I have confidence that it does the right thing.